packages/messaging/lib/index.d.ts

```diff
@@ -608,7 +608,7 @@ export namespace FirebaseMessagingTypes {
      * @param authorizedEntity The messaging sender ID. In most cases this will be the current default app.
      * @param scope The scope to assign a token, which the sever can use to target messages at.
      */
-    getToken(authorizedEntity?: string, scope?: string = 'FCM'): Promise<string>;
+    getToken(authorizedEntity?: string, scope?: string): Promise<string>;
```

Comment:
Interesting - this default string value you are removing has some relation to this issue: #3714

Reply:
Hey @mikehardy -- the place that the default `scope = 'FCM'` is set is in the JS code here: https://github.com/invertase/react-native-firebase/blob/master/packages/messaging/lib/index.js#L142-L159.

`.d.ts` definitions have no runtime responsibilities: parameter defaults are not allowed.

If changing the default scope is a change needed in #3714, it needs to happen in the source file.

Reply:
Yes definitely - I just thought it was interesting the 'FCM' scoped was baked in here as well, and since it's an active discussion item I was cross-linking for cross-pollination. I commented over there that I'd love a PR to do the actual change